hw/top_earlgrey/data/pins_cw310.xdc

```diff
@@ -21,37 +21,44 @@ set_property -dict { PACKAGE_PIN N21 IOSTANDARD LVCMOS33 } [get_ports { IO_CLK }
 #set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets IO_SDCK_IBUF]; # SDCK clock to be ignored
 
 ## LEDs
-set_property -dict { PACKAGE_PIN M26  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOA8 }]; #LED 0
-set_property -dict { PACKAGE_PIN M25  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB0 }]; #LED 1
-set_property -dict { PACKAGE_PIN M24  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB1 }]; #LED 2
-set_property -dict { PACKAGE_PIN M19  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB2 }]; #LED 3
-set_property -dict { PACKAGE_PIN L25  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB3 }]; #LED 4
-set_property -dict { PACKAGE_PIN K26  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB4 }]; #LED 5
-set_property -dict { PACKAGE_PIN L24  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB5 }]; #LED 6
-set_property -dict { PACKAGE_PIN K25  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOB6 }]; #LED 7
+set_property -dict { PACKAGE_PIN M26  DRIVE 8 IOSTANDARD LVCMOS33 } [get_ports { IOA0 }]; #LED 0
```

Comment:
Are these pad mappings arbitrary? Would we want to consider trying to get some overlap in features between the SAM3X and Hyperdebug?

Reply:
...I suppose the UART is the only other non-GPIO SAM3X function that is defined, so maybe there's not much to think about here.

Reply:
These are not entirely arbitrary. We still have some demos in our tree that use the switches and LEDs via GPIO, and I kept them for this particular XDC mapping since this is the "CW310 board centric" one (as opposed to "HYP centric" where most of the IOs are claimed by the HYP). 

We may be able to shuffle this around some more to free up the IOA0 and IOA1 pins, but if we want to keep the GPIO device LSBs aligned with the LEDs and switch bank, we need a more elaborate pinmux config (right now we just linearly map them, starting from IOA0).

I would say we iteratively do some more alignment (for things that make sense), once this has gone in.
The main motivation for this PR is getting the SW and TAP strap pins into the correct position and connect them to SAM3X so that the work on the new bootstrap protocol can proceed.

I will open an issue to track the alignments.

Reply:
That sounds good to me :+1: Thanks!

Reply:
Issue opened here: #12722